js/apps/system/_admin/aardvark/APP/frontend/js/models/arangoCollectionModel.js

```diff
@@ -334,22 +334,38 @@
         }
       });
     },
+    
+    changeComputedValues: function (computedValues, callback) {
+      if (!computedValues) {
+        computedValues = null;
+      }
+
+      $.ajax({
+        cache: false,
+        type: 'PUT',
+        url: arangoHelper.databaseUrl('/_api/collection/' + encodeURIComponent(this.get('id')) + '/properties'),
```

Comment:
It seems accessing collection names by their numeric ID has been deprecated since 3.4.0. Please see https://www.arangodb.com/docs/stable/http/collection-modifying.html#change-properties-of-a-collection.

Reply:
Fixed